tests/unit/parser/src/test_decode_manifest.c

```diff
@@ -0,0 +1,578 @@
+#include "suit_parser_test_utils.h"
+#include "suit_platform_mock_ext.h"
+
+static uint8_t cbor_envelope_empty_manifest[] = {
+	0xd8, 0x6b, /* tag(107) : SUIT_Envelope */
+	0xa2, /* map (2 elements) */
+
+	0x02, /* suit-authentication-wrapper */
+		0x42, /* bytes(2) */
+		0x81, /* array (1 element) */
+			0x40, /* bytes(36) */
+
+	0x03, /* suit-manifest */
+	0x40, /* bytes(0) */
+};
+
+static uint8_t cbor_envelope_no_common[] = {
+	0xd8, 0x6b, /* tag(107) : SUIT_Envelope */
+	0xa2, /* map (2 elements) */
+
+	0x02, /* suit-authentication-wrapper */
+		0x42, /* bytes(2) */
+		0x81, /* array (1 element) */
+			0x40, /* bytes(36) */
+
+	0x03, /* suit-manifest */
+	0x45, /* bytes(5) */
+	0xa2, /* map (2 elements) */
+	0x01, /* suit-manifest-version */ 0x00,
```

Comment:
is the suit-manifest-version set intentionally to 0 (0x00)?

Reply:
Nice catch! This is indeed a copy-paste issue.